templates/taxi/driver_detail.html

```diff
@@ -3,6 +3,7 @@
 {% block content %}
   <h1>
     Username: {{ driver.username }}
+    <a style="float: right" class="btn btn-primary" href="{% url 'taxi:driver-update' driver.id %}">Update license</a>
```

Comment:
Ensure that the `url` tag syntax is correct when passing arguments. It should be `{% url 'taxi:driver-update' driver.id %}`.